src/ManageCourses.ApiClient/ManageCourses.ApiClient.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="packages\GitInfo.2.0.3\build\GitInfo.targets" Condition="Exists('packages\GitInfo.2.0.3\build\GitInfo.targets')" />
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <PackageId>GovUk.Education.ManageCourses.ApiClient</PackageId>
+    <Authors>Department for Education</Authors>
+    <Description>Client library to access the "Manage Courses" api</Description>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <PackageReleaseNotes>Pre-release</PackageReleaseNotes>
+    <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
+    <GitInfoReportImportance>high</GitInfoReportImportance>
+    <GitVersionFile>$(ProjectDir)GitVersion.txt</GitVersionFile>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RootNamespace>GovUk.Education.ManageCourses.ApiClient</RootNamespace>
+  </PropertyGroup>
+  <PropertyGroup>
+    <OutputSwaggerGeneration Condition="$(OutputSwaggerGeneration) == ''">Generated/ManageCoursesApiClient.cs</OutputSwaggerGeneration>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="$(OutputSwaggerGeneration)" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="GitInfo" Version="2.0.11" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.0.2" />
+    <PackageReference Include="NSwag.MSBuild" Version="11.17.13" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\ManageCourses.Api\ManageCourses.Api.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Update="integration-tests.json" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
Is there a reason?

Reply:
Yes - Including the package reference ensures that the ManageCouses.Api project gets built and is added to `@(_ResolvedProjectReferencePaths, ',')`

Reply:
Oh did you mean the `integration-tests.json` line - sorry I didn't spot that. Yeah that needs to go. Thx